applications/RomApplication/tests/fluid_dynamics_test_files/test_ROM.py

```diff
@@ -1,8 +1,13 @@
 from __future__ import print_function, absolute_import, division
 import KratosMultiphysics
-import numpy as np
 
-from .MainKratosROM import TestFluidDynamicsROM
+try:
+    import numpy as np
+    from .MainKratosROM import TestFluidDynamicsROM
```

Comment:
to indicate if a test needs apps you should use
https://github.com/KratosMultiphysics/Kratos/blob/37b3103c2380c5bcbc18c9ae094b18114285cc48/kratos/python_scripts/KratosUnittest.py#L38
or
https://github.com/KratosMultiphysics/Kratos/blob/37b3103c2380c5bcbc18c9ae094b18114285cc48/kratos/python_scripts/KratosUnittest.py#L106

Reply:
Now using this method for skipping if applications are not compiled